qemu/tests/nic_hotplug.py

```diff
@@ -233,6 +250,6 @@ def device_add_nic(pci_model, netdev, device_id):
             vm.set_link(nic.device_id, up=True)
         logging.info("Reboot vm to verify it alive after hotunplug nic(s)")
         serial = len(vm.virtnet) > 0 and False or True
-        session = vm.reboot(serial=serial)
-        vm.verify_alive()
-        session.close()
+        reboot_vm(vm=vm, need_session=False, serial=serial)
```

Comment:
too many options in finally, the error happen in finally will cover the error in the testing

Reply:
Updated , thanks